consensus/istanbul/backend/engine_test.go

```diff
@@ -80,7 +80,8 @@ func makeCommittedSeals(hash common.Hash) [][]byte {
 func newBlockChain(n int, items ...interface{}) (*blockchain.BlockChain, *backend) {
 	// generate a genesis block
 	genesis := blockchain.DefaultGenesisBlock()
-	genesis.Config = params.TestChainConfig
+	config := *params.TestChainConfig // copy test chain config which may be modified
+	genesis.Config = &config
```

Comment:
It seems good to import chainconfig like this way in case the chainconfig changes later. LGTM